packages/react-dom/src/events/__tests__/SyntheticEvent-test.js

```diff
@@ -18,19 +17,23 @@ describe('SyntheticEvent', () => {
   var createEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticEvent = require('events/SyntheticEvent').default;
     React = require('react');
     ReactDOM = require('react-dom');
     ReactTestUtils = require('react-dom/test-utils');
 
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget').default(nativeEvent);
-      return SyntheticEvent.getPooled({}, '', nativeEvent, target);
+    createEvent = (nativeEvent, callback) => {
+      var instance;
+      var container = document.createElement('div');
```

Comment:
I would like to use the same beforeEach/afterEach structure as in https://github.com/facebook/react/pull/11365. Set up a container and add it to the document, then remove it.